readthedocs/core/unresolver.py

```diff
@@ -52,10 +52,11 @@ def __init__(self, project, version_slug, filename):
 
 
 class TranslationNotFoundError(UnresolverError):
-    def __init__(self, project, language, filename):
+    def __init__(self, project, language, filename, version_slug):
         self.project = project
         self.language = language
         self.filename = filename
+        self.version_slug = version_slug
```

Comment:
Should this just be called `version`? Are the other variables here not slugs?

Reply:
We are using `project` and `version` in several parts of proxito, so being explicit that this isn't an object, but a slug, makes sense to me. For this particular case, we don't have access to the full version (it doesn't exist), so we just have the slug.

Reply:
👍🏼 to add that comment here.

Also, it makes sense to reorder the arguments to keep consistency with the other exceptions. Usually it comes `project, version_slug`, but this one is different.